server/world/sound/block.go

```diff
@@ -188,6 +188,24 @@ type PotionBrewed struct{ sound }
 // LecternBookPlace is a sound played when a book is placed in a lectern.
 type LecternBookPlace struct{ sound }
 
+// RespawnAnchorCharge is a sound played when a respawn anchor has been charged.
+type RespawnAnchorCharge struct {
+	Charge int32
+	sound
```

Comment:
Please embed sound before the Charge field for this struct and the two below, i'd also make Charge an int and cast it where necessary